db/system_distributed_keyspace.cc

```diff
@@ -96,8 +97,27 @@ schema_ptr cdc_desc() {
         return schema_builder(system_distributed_keyspace::NAME, system_distributed_keyspace::CDC_DESC, {id})
                 /* The timestamp of this CDC generation. */
                 .with_column("time", timestamp_type, column_kind::partition_key)
-                /* The set of stream identifiers used in this CDC generation. */
+                /* For convenience, the list of stream IDs in this generation is split into token ranges
+                 * which the stream IDs were mapped to (by the partitioner) when the generation was created.  */
+                .with_column("range_end", long_type, column_kind::clustering_key)
+                /* The set of stream identifiers used in this CDC generation for the token range
+                 * ending on `range_end`. */
                 .with_column("streams", cdc_streams_set_type)
+                .with_version(system_keyspace::generate_schema_version(id))
+                .build();
```

Comment:
It looks like you're changing the schema of CDC_DESC. That can't be right.

If you changed the contents of system_distributed_keyspace::CDC_DESC, that's just confusing.

Reply:
You mean I should also change the variable name to e.g. `CDC_DESC_V2`?

Reply:
Yes.